agasc/supplement/magnitudes/mag_estimate_report.py

```diff
@@ -297,12 +299,14 @@ def multi_star_html(
             agasc_stats["last_obs"] = CxoTime(agasc_stats["last_obs_time"]).date
 
         if len(updated_stars):
-            agasc_stats["update_mag_aca"][
-                np.in1d(agasc_stats["agasc_id"], updated_star_ids)
-            ] = updated_stars["mag_aca"]
-            agasc_stats["update_mag_aca_err"][
-                np.in1d(agasc_stats["agasc_id"], updated_star_ids)
-            ] = updated_stars["mag_aca_err"]
+            in_agasc_stats = np.in1d(updated_stars["agasc_id"], agasc_stats["agasc_id"])
+            if np.any(~in_agasc_stats):
+                raise ValueError(
+                    "ome stars in updated_stars are not not in agasc_stats"
+                )
```

Comment:
There is a typo, but more importantly, is this new exception caught by the calling code so that the processing will not fail? Or if it does, is there a procedure to fix whatever happened to cause the exception?

Reply:
This new exception replaces an exception that would be raised by the code being replaced (trying an assignment with mismatching arrays):
```
            agasc_stats["update_mag_aca"][
                np.in1d(agasc_stats["agasc_id"], updated_star_ids)
            ] = updated_stars["mag_aca"]
```

Reply:
Fixed typo.

Reply:
Tom, this behaviur has been there since age immemorial, I'm just keeping it so it continues to raise in the same circumstance.

The code that calls this function creates the `updated_stars` so this effectively never happens. If it were to happen, the whole script will fail, **as it should**.

This check is here in case one calls the function directly, which I have done for debugging purposes on occasion.

Reply:
Also, this code is called to make the report after the supplement is made.

If you want me to "fix" this, I can do it, but you are requesting me to fix a non-issue that has been there forever, and I still do not know what the fix will be. Do you really think that is what I should be doing?

Reply:
You did not answer my question of "what do we do if this DOES actually happen.". But that's OK, please just add a sentence to the exception message like "This should never happen, if it does then some debugging is needed."